peps/pep-0007.rst

```diff
@@ -135,6 +135,20 @@ Code lay-out
           return 0; /* "Forgive" adding a __dict__ only */
       }
 
+* Vertically align line continuation characters in multi-line macros.
+
+* Macros intended to be used as a statement should use the
+  ``do {... } while(0)`` macro idiom.
+  Example::
+
+      #define ADD_INT_MACRO(MOD, INT) do {                          \
```

Comment:
I prefer to put do{ on a separated line so the macro is at a different indent than the regular code, for macros defined inside functions.

Reply:
I think it's ok to not explicitly recommend a particular style here.

Reply:
Addressed in 31be7f224cec8d6b09a5aa20d913c87a9384e506